README.md

```diff
@@ -23,6 +23,14 @@ Type the path to the output to run the program.
 
 After running the program generates a file name TimeSheet.txt, this is where you copy the inputed hours from.
 
+## Usage
+
+- The first thing it will ask you is the month. Input the month number and not the name.
+- Now enter the date for sunday of the week, and just the day of the month nothing else.
+- Now you can go on with entering your time blocks starting Sunday until Saturday.
+- It asks you for hours given to respective timeblocks in order.
```

Comment:
This is pretty unclear to me. As someone who's never (and will never) use this app, when I understand it, anyone can!
Basically the "to respective timeblocks" is unclear. And "timeblocks" is not a word.